etl/steps/data/garden/tsmc/2025-11-10/operating_data.meta.yml

```diff
@@ -0,0 +1,156 @@
+          <% elif category == "technology" and metric == "16nm" %>
+          The share of TSMC's revenue from 16nm chips, used in GPUs, networking, and automotive applications.
+          <% elif category == "technology" and metric == "20nm" %>
+          The share of TSMC's revenue from legacy 20nm chips.
+          <% elif category == "technology" and metric == "28nm" %>
+          The share of TSMC's revenue from 28nm chips, widely used in automotive, IoT, and industrial applications.
+          <% elif category == "technology" and metric == "40/45nm" %>
+          The share of TSMC's revenue from 40/45nm chips, used in automotive and industrial applications.
+          <% elif category == "technology" and metric == "65nm" %>
+          The share of TSMC's revenue from 65nm chips, used in automotive and industrial applications.
+          <% elif category == "technology" and metric == "90nm" %>
+          The share of TSMC's revenue from 90nm chips, used in power management and display drivers.
+          <% elif category == "technology" and metric == "0.11/0.13um" %>
+          The share of TSMC's revenue from 110/130nm chips, used in long-lifecycle industrial and automotive products.
+          <% elif category == "technology" and metric == "0.15um" %>
+          The share of TSMC's revenue from 150nm chips, used in specialized industrial and analog applications.
+          <% elif category == "technology" and metric == "0.18um" %>
+          The share of TSMC's revenue from 180nm chips, used in power management and analog circuits.
+          <% elif category == "technology" and metric == "0.25um+" %>
+          The share of TSMC's revenue from 250nm and larger chips, used in specialized analog and industrial applications.
+          <% elif category == "platform" and metric == "High Performance Computing" %>
+          The share of TSMC's revenue from high-performance computing, including AI accelerators and data center processors.
+          <% elif category == "platform" and metric == "Smartphones" %>
+          The share of TSMC's revenue from smartphone processors and mobile chipsets.
+          <% elif category == "platform" and metric == "Internet of Things" %>
+          The share of TSMC's revenue from Internet of Things applications, including smart home devices and wearables.
+          <% elif category == "platform" and metric == "Automotive" %>
+          The share of TSMC's revenue from automotive applications, including ADAS and infotainment systems.
+          <% elif category == "platform" and metric == "Digital Consumer Electronics" %>
+          The share of TSMC's revenue from consumer electronics like TVs and gaming consoles.
+          <% elif category == "platform" and metric == "Others" %>
+          The share of TSMC's revenue from other applications not classified in major platform categories.
+          <% elif category == "geography" %>
+          The share of TSMC's revenue from << metric >>.
```

Comment:
Looks like you could have a first top-level if that goes `if category in {"platform", "technology", "geography"}` and evaluates to string "The share of TSMC's revenue from". Then add second-level if to add the ending text part.

Reply:
And I think that other fields in the metadata could benefit from this pattern